torch/_torch_docs.py

```diff
@@ -7298,7 +7309,8 @@ def merge_dicts(*dicts):
     :func:`torch.tensor` always copies :attr:`data`. If you have a Tensor
     ``data`` and want to avoid a copy, use :func:`torch.Tensor.requires_grad_`
     or :func:`torch.Tensor.detach`.
-    If you have a NumPy ``ndarray`` and want to avoid a copy, use
+    If you have a NumPy ``ndarray`` or an object that implements
```

Comment:
"If :attr:\`data\` is a Python object that implements the CPU Array Interface, like a NumPy array does, or the CUDA Array Interface, then see :func:\`torch.as_tensor\` for how to create a tensor that shares storage with :attr:\`data\` and avoid this copy.

Be sure to add links for both CPU Array Interface and CUDA Array Interface here, too, like in the above.